src/resources/views/buttons/delete.blade.php

```diff
@@ -28,6 +28,10 @@ function deleteEntry(button) {
 	                  // Hide the modal, if any
 	                  $('.modal').modal('hide');
 
+	                  // Remove the details row, if it is open
+	                  if(row.hasClass("shown"))
```

Comment:
Space after `if` - and please use braces ('{}`) even for one line `then` clauses.

Reply:
Thank you for the review @lloy0076, it's fixed!